app/views/question_flow/applicant.html.slim

```diff
@@ -15,7 +15,7 @@
       li.govuk-hint = _1
 
 .govuk-grid-column-two-thirds
-  = form_for(@form, url: wizard_path, method: :put) do |form|
+  = form_for(@form, url: request.path, method: :put) do |form|
```

Comment:
So I _believe_ the difference is that fullpath includes the query string if their is one. We don't use the query string, and including it could introduce complications (what if there's both a form field with `name="foo" value="bar"` and `?foo=baz` in the querystring? Does `params[:foo]` have value `bar` or `baz`?), so just using the path keeps things simple.